flutter/shell/platform/tizen/nui_autofill_popup.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_NUI_AUTOFILL_POPUP_H_
+#define EMBEDDER_NUI_AUTOFILL_POPUP_H_
+
+#include <dali-toolkit/devel-api/controls/popup/popup.h>
+
+#include <functional>
+
+namespace flutter {
+
+using OnCommit = std::function<void(std::string str)>;
+using OnRendering = std::function<void()>;
+
+class NuiAutofillPopup : public Dali::ConnectionTracker {
```

Comment:
Why does this class inherit from `Dali::ConnectionTracker`?

Reply:
The reason for inheriting `Dali::ConnectionTracker` in `NuiAutofillPopup` is to use `Connect` function in `Dali::Signal`.